src/r2mm/profiles/ProfilesClient.ts

```diff
@@ -19,6 +19,19 @@ function formatApiError<T>(e: T, genericTitle: string): R2Error | T {
                 message,
                 "You were rate limited by the server, wait for a while and try again."
             );
+        } else if (e.response.status === 404) {
+            // Note that this error text is only applicable when importing, but it should be OK as creating a profile shouldn't respond with 404 anyway.
+            return new R2Error(
+                genericTitle,
+                `Server responded with \"404: Not Found\" while trying to download the profile`,
```

Comment:
Nitpick: template literals (using backticks when defining a string) are usually used only when embedded expression (`${myVariable}`) is used. Often a linting tool enforces such behaviour. But linting is not currently present, so I don't consider that an enforceable rule. The code base already mixes `"` and `'`, so I wouldn't want to add a third option into the mix, though.

(Also if you end up using the backticks anyway, you don't need to escape quotation marks inside them.)